lib/elixir/lib/process.ex

```diff
@@ -544,22 +546,34 @@ defmodule Process do
 
   Inlined by the compiler.
   """
-  @spec flag(process_flag, term) :: term
+  @spec flag(:error_handler, module) :: module
+  @spec flag(:max_heap_size, heap_size) :: heap_size
+  @spec flag(:message_queue_data, :erlang.message_queue_data) :: :erlang.message_queue_data
```

Comment:
Sorry a poor choice of words, I meant in the docs for the function because it is not obvious to a user that those flags only work in OTP 19.